### DISCUSSION TITLE
Area picker questions

### DISCUSSION THREAD

**Discussion by rozekmichal** [Cediner/ArdClient#91] [Q&A]
Hey, 

Is there any guide how to properly use area picker? It's working for me for the most of the time while I'm gathering branches, bark from the trees etc but I don't know how to automate stumps destroying. 

What I do is:
1) Select the area
2) add all stumps to gob list
3) choose action to: `Action:Destroy`
4) Tried all storage types but it doesn't matter for me here

I was having enough energy and a shovel in my hands. I was able to destroy stumps manually. 
My character is not moving and after a couple of seconds I've got "Finished" status.

Thanks for giving us this beautiful client <3

**Comment by Cediner** [comment]
this option is under development

**Reply by rozekmichal** [reply]
Ok, cool. Thanks for doing this!
